src/main/kotlin/org/jitsi/nlj/RtpReceiverImpl.kt

```diff
@@ -88,6 +89,7 @@ class RtpReceiverImpl @JvmOverloads constructor(
      */
     private val backgroundExecutor: ScheduledExecutorService,
     streamInformationStore: ReadOnlyStreamInformationStore,
+    private val eventHandler: TransceiverEventHandler,
```

Comment:
Passing `TransceiverEventHandler` feels a little weird here.  I was wondering if we defined an `RtpReceiverEventHandler`, then we couild have a `TransceiverEventHandler` which extended the `RtpReceiverEventHandler` interface so we wouldn't have to duplicate the code and would get the separation...but I don't feel strongly about it so I'm fine with it as-is.